src/components/enterprise-user-subsidy/SubscriptionSubsidy.jsx

```diff
@@ -12,15 +11,13 @@ import {
 import { useRenderContactHelpText } from '../../utils/hooks';
 import { LICENSE_STATUS } from './data/constants';
 
-const SubscriptionSubsidy = ({ plan, license }) => {
-  const { enterpriseConfig } = useContext(AppContext);
+const SubscriptionSubsidy = ({ enterpriseConfig, plan, license }) => {
```

Comment:
I may have commented to this effect, but we could now pull all these props out of the relevant context instead since this is rendered as a child of `UserSubsidy` now.

Reply:
I prefer to have this component be purely display, and let it's parent component handle the data. That's not *completely* the case here, since it still has a lot of logic based on it's data, but I like the separation of concerns, especially if we're moving towards a system that's less a permanent display of alerts and more like your Toast system.